packages/desmos-std/src/profiles/querier.rs

```diff
@@ -0,0 +1,62 @@
+use crate::{
+    profiles::{
+        models_app_links::{
+            QueryApplicationLinkByClientIDResponse, QueryApplicationLinksResponse,
+            QueryUserApplicationLinkResponse,
+        },
+        models_blocks::QueryBlocksResponse,
+        models_chain_links::{QueryChainLinksResponse, QueryUserChainLinkResponse},
+        models_dtag_requests::QueryIncomingDtagTransferRequestResponse,
+        models_profile::QueryProfileResponse,
+        models_relationships::QueryRelationshipsResponse,
+    },
+    types::PageRequest,
+};
+use cosmwasm_std::{Addr, StdResult, Uint64};
+
+pub trait ProfilesQuerier {
```

Comment:
Could we build querier for each modules individually? I think we can create querier from the `cosmos_std::Deps` instead of creating our `Deps` with fixed query type.
Just tried it with the following branch:
https://github.com/desmos-labs/desmos-contracts/blob/paul/subspaces-queries-clean/packages/desmos-std/src/subspaces/querier.rs#L18

Reply:
As long as it simplifies devs' works it's fine. Plus, it will let us to better organise code base on each module independently from one module to another